clean.cmd

```diff
@@ -1,163 +1,31 @@
-  set clean_targets=Clean;CleanPackages;CleanPackagesCache;
-  set clean_all=true
-  set thisArgs=!thisArgs!%1
-  goto Next
-)
-
-if [!thisArgs!]==[] (
-  set unprocessedBuildArgs=!allargs!
-) else (
-  call set unprocessedBuildArgs=%%allargs:*!thisArgs!=%%
-)
-
-:Next
-shift /1
-goto Loop
-
-:Begin
-if /I [%clean_environment%] == [true] (
-  call :CleanEnvironment
-)
-
-if /I [%clean_src%] == [true] (
-  echo Cleaning src directory ...
-  echo. >> %cleanlog% && echo git clean -xdf %~dp0src >> %cleanlog%
-  call git clean -xdf %~dp0src >> %cleanlog%
-  call :CheckErrorLevel
-)
-
-if NOT "%clean_targets%" == "" (
-  echo Running init-tools.cmd
-  call %~dp0init-tools.cmd
-  
-  echo Running msbuild clean targets "%clean_targets:~0,-1%" ...
-  echo. >> %cleanlog% && echo msbuild.exe %~dp0build.proj /t:%clean_targets:~0,-1% !options! !unprocessedBuildArgs! >> %cleanlog%
-  call msbuild.exe %~dp0build.proj /t:%clean_targets:~0,-1% !options! !unprocessedBuildArgs!
-  call :CheckErrorLevel
-)
-
-if /I [%clean_tools%] == [true] (
-  call :CleanEnvironment
-  echo Cleaning tools directory ...
-  echo. >> %cleanlog% && echo rmdir /s /q %~dp0tools >> %cleanlog%
-  rmdir /s /q %~dp0tools >> %cleanlog%
-  REM Don't call CheckErrorLevel because if the Tools directory didn't exist when this script was
-  REM invoked, then it sometimes exits with error level 3 despite successfully deleting the directory.
-)
+echo Stop VBCSCompiler.exe execution.
+for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.exe" /fo csv') do taskkill /f /PID %%~F
 
-if /I [%clean_all%] == [true] (
-  call :CleanEnvironment
+if [%1] == [-all] (
   echo Cleaning entire working directory ...
-  echo. >> %cleanlog% && echo git clean -xdf -e clean.log %~dp0 >> %cleanlog%
-  call git clean -xdf -e clean.log %~dp0 >> %cleanlog%
-  call :CheckErrorLevel
+  call git clean -xdf
+  exit /b !ERRORLEVEL!
 )
 
-if /I [%clean_successful%] == [true] (
-  echo Clean completed successfully.
-  echo. >> %cleanlog% && echo Clean completed successfully. >> %cleanlog%
-  exit /b 0
-) else (
-  echo An error occured while cleaning; see %cleanlog% for more details.
-  echo. >> %cleanlog% && echo Clean completed with errors. >> %cleanlog%
-  exit /b 1
-)
+if [%1]==[] set __args=-b
+call %~dp0run.cmd clean %__args% %*
+exit /b %ERRORLEVEL%
 
 :Usage
 echo.
 echo Repository cleaning script.
 echo.
 echo Options:
-echo     /b     - Deletes the binary output directory.
-echo     /p     - Deletes the repo-local nuget package directory.
-echo     /c     - Deletes the user-local nuget package cache.
-echo     /t     - Deletes the tools directory.
-echo     /s     - Deletes the untracked files under src directory (git clean src -xdf).
-echo     /e     - Kills and/or stops the processes that are still running, for example VBCSCompiler.exe
-echo     /all   - Combines all of the above.
+echo     -b     - Deletes the binary output directory. It uses the Run Command Tool to do it.
```

Comment:
Unless I'm missing something it doesn't look like the usage is coming from run, it is still in the script.